config/v1/types_cluster_version.go

```diff
@@ -749,3 +780,26 @@ type ClusterVersionList struct {
 
 	Items []ClusterVersion `json:"items"`
 }
+
+// SignatureStore represents the URL of custom Signature Store
+type SignatureStore struct {
+
+	// url contains the upstream custom signature store URL.
+	// url should be a valid absolute http/https URI of an upstream signature store as per rfc1738.
+	// This must be provided and cannot be empty.
+	//
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:XValidation:rule="isURL(self)",message="url must be a valid absolute URL"
+	// +required
```

Comment:
This tag is incorrect and not recognised, hence the errors you're getting saying the list map key is not required.
```suggestion
	// +kubebuilder:validation:Required
```